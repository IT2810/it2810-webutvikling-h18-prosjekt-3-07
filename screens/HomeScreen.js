import React from 'react';
import {View, StyleSheet } from 'react-native';
import NavigationButton from '../components/NavigationButton';

class HomeScreen extends React.Component {
    static navigationOptions= {
      title: 'Home',
    };
    render() {
      return (
        <View style={styles.container}>
          <NavigationButton 
            onPress={() => {
              this.props.navigation.navigate('Tasks');
            }}
            title = "TODO'S"
          />
          <NavigationButton 
            onPress={() => {
              this.props.navigation.navigate('Pedometer');
            }}
            title = "Pedometer"
          />
        </View>
      );
    }
  }

  const styles = StyleSheet.create({
    container: {
      flex: 1,
      alignItems: "center",
      justifyContent: "center",
      backgroundColor: '#fff',
    }
  });

  export default HomeScreen;
