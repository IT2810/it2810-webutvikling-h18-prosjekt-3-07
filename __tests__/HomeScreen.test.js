import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import HomeScreen from "../screens/HomeScreen";

it("matches snapshot", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<HomeScreen />);
  expect(result).toMatchSnapshot();
});
