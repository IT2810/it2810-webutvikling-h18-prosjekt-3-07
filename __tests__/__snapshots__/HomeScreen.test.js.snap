// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <NavigationButton
    onPress={[Function]}
    title="TODO'S"
  />
  <NavigationButton
    onPress={[Function]}
    title="Pedometer"
  />
</Component>
`;
